### ISSUE TITLE
When user has a pool VM created from template in cluster that user doesn't have any permissions for, the template name for the VM is N/A instead of the name of the template.

### ISSUE THREAD

**Issue by sjd78** [oVirt/ovirt-web-ui#1058]
When user has a pool VM created from template in cluster that user doesn't have any permissions for, the template name for the VM is N/A instead of the name of the template.

_Originally posted by @isaranova in https://github.com/oVirt/ovirt-web-ui/pull/1019#issuecomment-504404351_

**Comment by bamsalem** [comment]
Hi Guys I checked that and it seems like the issue is already fixed:
I logged in to the web Admin as an admin and removed t1 user's permissions to Default-new cluster:
cluster's permissions screenshot:
![permissions](https://user-images.githubusercontent.com/64131213/91823550-0b4fd780-ec42-11ea-91a3-f1709040d730.png)
Then I logged in to vmPortal as t1 user and viewed pool VM which created from template in Default-new cluster.
The described issue wasn't occur:
vmDetails card screenshot:
![image](https://user-images.githubusercontent.com/64131213/91823524-01c66f80-ec42-11ea-8ce3-9a37058eecca.png)

It seems it's already been fixed.
did I checked it well? or I did something wrong in the proccess?
@sgratch @sjd78
